@proy30 a small hiccup that I see in the dashboard test in CI: [job-logs.txt](https://github.com/user-attachments/files/21825631/job-logs.txt)

```
>       raise TimeoutError(
            f"state['{state_name}'] never became '{expected_input}' after {timeout} seconds "
            f"(last value: '{value}').\n"
            f"curr_view_details_log: {self.get_state('curr_view_details_log')}"
        )
E       TimeoutError: state['total_steps'] never became '125' after 60 seconds (last value: '101').
E       curr_view_details_log:

/home/runner/work/impactx/impactx/tests/python/dashboard/utils.py:226: TimeoutError
=========================== short test summary info ============================
FAILED ../../../tests/python/dashboard/test_dashboard.py::test_dashboard - TimeoutError: state['total_steps'] never became '125' after 60 seconds (last value: '101').
curr_view_details_log:
=================== 1 failed, 2 passed in 108.69s (0:01:48) ====================
```

Is this something you might have an idea how to make more robust?
Update: moved to #1107

Restarting CI for now :)

--- a/tests/python/test_charge_deposition.py
+++ b/tests/python/test_charge_deposition.py
@@ -50,7 +50,7 @@ def test_charge_deposition(save_png=True):
     dV = np.prod(dr)
 
     beam_charge = dV * rs  # in C
-    assert math.isclose(beam_charge, -1.0e-9, rel_tol=1.0e-8)
+    assert math.isclose(beam_charge, -1.0e-9, rel_tol=8.8e-7)
 
     half_z = sim.n_cell[2] // 2  # order: x,y,z
 

--- a/tests/python/test_charge_deposition.py
+++ b/tests/python/test_charge_deposition.py
@@ -50,7 +50,7 @@ def test_charge_deposition(save_png=True):
     dV = np.prod(dr)
 
     beam_charge = dV * rs  # in C
-    assert math.isclose(beam_charge, -1.0e-9, rel_tol=8.8e-7)
+    assert math.isclose(beam_charge, -1.0e-9, rel_tol=9.8e-7)
 
     half_z = sim.n_cell[2] // 2  # order: x,y,z
 

--- a/tests/python/test_charge_deposition.py
+++ b/tests/python/test_charge_deposition.py
@@ -13,7 +13,7 @@ import matplotlib.pyplot as plt
 import numpy as np
 from conftest import basepath
 
-from impactx import ImpactX, amr
+from impactx import Config, ImpactX, amr
 
 
 def test_charge_deposition(save_png=True):
@@ -50,7 +50,10 @@ def test_charge_deposition(save_png=True):
     dV = np.prod(dr)
 
     beam_charge = dV * rs  # in C
-    assert math.isclose(beam_charge, -1.0e-9, rel_tol=9.8e-7)
+    if Config.precision == "SINGLE":
+        assert math.isclose(beam_charge, -1.0e-9, rel_tol=1e-6)
+    else:
+        assert math.isclose(beam_charge, -1.0e-9, rel_tol=1e-8)
 
     half_z = sim.n_cell[2] // 2  # order: x,y,z
 

